docs/source/background/scf_notes.md

```diff
@@ -0,0 +1,86 @@
+<!--
+  ~ Copyright 2025 NWChemEx-Project
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+-->
+
+# SCF Procedure
+
+## SCF Driver
+
+1. Obtain the AO parameters as well as the chemical system (inputs)
+2. Assign the AO parameters as atomic orbitals
```

Comment:
I assume this is referring to a line where we go from an `AOBasisSet` to an `AOSpace`. There's no physical meaning behind that. That's just a type conversion.

Reply:
IMHO, step 2 should read something like:

```
Mathematically, the inputs to the SCF procedure are a trial wavefunction and a Hamiltonian. Here, the input `ChemicalSystem` is converted to a Hamiltonian by calling a module of property type `WhateverPT`. By default, this is done subject to the Born-Openheimer approximation (i.e., the resulting Hamiltonian will contain the pair-wise interactions of all the particles, but only the kinetic energy of the electrons). 
```
In my mind, you wouldn't go into further details of the "Hamiltonian" module here (that would be reserved for the module documentation of "Hamiltonian" and linked to here).